test/test_citusdb_baseRI.yml

```diff
@@ -0,0 +1,165 @@
+---
+- hosts: cdbc01
+  name: Citus RealTime Analytics Test
+  tasks:
+    - name: Download users data (31.7MB)
+      get_url:
+        url: https://examples.citusdata.com/tutorial/users.csv
+        dest: /tmp/users.csv
+        mode: 0755
+
+    - name: Download events data (74.7MB)
+      get_url:
+        url: https://examples.citusdata.com/tutorial/events.csv
+        dest: /tmp/events.csv
+        mode: 0755
+
+    - name: "create table users"
+      become: yes
+      become_user: postgres
+      become_method: sudo
+      command: >
+        psql -c "create table github_users(user_id bigint,url text,login text,avatar_url text,
+        gravatar_id text,display_login text);"
```

Comment:
@lel99999 `become_user` doesn't work well with our infrastructure for some reason, so it is best to avoid it.  The alternative would be to use the `postgresql_exec` module I created, which allows you to pass `pp_superadmin` and `pp_superpassword` as parameters, and not use `psql`.  Please update this one and all those below.